lib/net.js

```diff
@@ -1112,6 +1112,7 @@ function Server(options, connectionListener) {
   }
 
   this._connections = 0;
+  this._lookup_state = {running: false};
```

Comment:
I think a `Symbol` would still be preferable here. Also, an object that only ever has a single property seems a little wasteful.